openvm/src/powdr_extension/executor/periphery.rs

```diff
@@ -18,28 +18,29 @@ pub struct PowdrPeripheryInstances {
 
 #[derive(Clone)]
 pub struct SharedPeripheryChips {
-    pub bitwise_lookup_8: SharedBitwiseOperationLookupChip<8>,
+    pub bitwise_lookup_8: Option<SharedBitwiseOperationLookupChip<8>>,
```

Comment:
This change was introduced because technically we can have an SdkVm without a lookup chip, see test for export_pil